app/config/scopes.php

```diff
@@ -1,6 +1,12 @@
 <?php
 
 return [ // List of publicly visible scopes
+    'account' => [
+        'description' => 'Access to make actions on behalf of a user account',
+    ],
```

Comment:
Can you give me some reasons why this is not `account.read` and `account.write`

Reply:
There was an existing hidden scope called `account`.
I wanted keep this name to reduce the scope of the changes.

Reply:
Changed